test/e2e/util.go

```diff
@@ -136,3 +139,17 @@ func validateInferenceWithClient(t *testing.T, client *openai.Client, modelName
 	assert.NotEmpty(t, chatCompletion.Choices, "chat completion has no choices returned")
 	assert.NotNil(t, chatCompletion.Choices[0].Message.Content, "chat completion has no message returned")
 }
+
+func validateAllPodsAreReady(t *testing.T, client *kubernetes.Clientset, expectedPodCount int) {
```

Comment:
i am curious we already check pod status here https://github.com/vllm-project/aibrix/blob/bc6cef162416496dd84ed6972c5eecff1fcc48a8/.github/workflows/installation-tests.yml#L142 and then run `make test-e2e`,  why does it work as expected and we have to check one more time here?

Reply:
In model adapter e2e test, to validate re-scheduling a pod on which lora adapter is scheduled is deleted. This validation is to ensure the deleted pod is ready. Hope this helps, this check is different from pod wait you have referenced.

Reply:
https://github.com/vllm-project/aibrix/blob/main/test/e2e/model_adapter_test.go#L65

Reply:
make sense.